marlowe-runtime/doc/ReadMe.md

```diff
@@ -1,100 +1,141 @@
-	- Retrieve the next transactions for a contract instance on the blockchain after a specified point on the blockchain.
-	- Follow the history of a contract instance.
-	- Retrieve the current status of all of the contract instances being followed.
+### Marlowe History (`marlowe-history`) 
+
+- provides services for querying the on-chain history of Marlowe contracts, 
+- uses `chainseekd` to follow the progress of a set of Marlowe contract instances, 
+- handles rollbacks in the blockchain.
+- 3 APIs, each on different TCP sockets.
+	- *job* API:
+		- Start following the history of a contract instance.
+		- Stop  following the history of a contract instance.
+	- *query* API :
+		- Fetch the status of all contracts followed by the service.
+		- Fetch the status of contracts in a provided set of contract identifiers.
+	- *sync* API :
+		- Find a contract instance on the blockchain, starting from the genesis block.
+		- Find a contract instance on the blockchain, starting from a list of intersection points on the blockchain.
+		- Retrieve the next transactions for a contract instance on the blockchain after a specified point on the blockchain.
+		- Follow the history of a contract instance.
+		- Retrieve the current status of all of the contract instances being followed.
 
 See `marlowe-history`'s [help page](marlowe-history.md) for more information.
 
 
-### Marlowe Discovery
+### Marlowe Discovery (`marlowe-discovery`)
 
-The `marlowe-discovery` executable provides services for discoverying the on-chain presence of Marlowe contracts. It uses `chainseekd` to detect the presence of contract instance of known versions of Marlowe, and it handles rollbacks in the blockchain.
 
-Two APIs, accessible over two different TCP sockets, are provided:
-- The *query* API supports two types of queries:
-	- Fetch the contract information for all Marlowe contract instances.
-	- Fetch the contract information for all Marlowe contract instances that use a particular currency symbol for their role tokens.
-- The *sync* API provides two capabilities:
-	- Retrieve the contract information for all Marlowe contracts after a specified point on the blockchain.
-	- Retrieve the most recent valid block header from a given list of intersection points on the blockchain.
+- provides services for discoverying the on-chain presence of Marlowe contracts. 
+- uses `chainseekd` to detect the presence of contract instance of known versions of Marlowe,
+- handles rollbacks in the blockchain.
+- 2 APIs, each on different TCP sockets.
+	- *query* API:
+		- Fetch the contract information for all Marlowe contract instances.
+		- Fetch the contract information for all Marlowe contract instances that use a particular currency symbol for their role tokens.
+	- *sync* API:
+		- Retrieve the contract information for all Marlowe contracts after a specified point on the blockchain.
+		- Retrieve the most recent valid block header from a given list of intersection points on the blockchain.
```

Comment:
It is also used to receive notifications when new Marlowe contracts are published and when they are rolled back

Reply:
replaced it with : 

```
	- *sync* API:
		- Retrieve the contract information for all Marlowe contracts after a specified point on the blockchain.
		- Retrieve the most recent valid block header from a given list of intersection points on the blockchain.
		- Used to receive notifications when new Marlowe contracts are published and when they are rolled back
```